libs/sdk-ui-filters/styles/scss/attributeFilterNext/attributeFilterDropdownButton.scss

```diff
@@ -203,7 +203,14 @@
     }
 }
 
+.gd-attribute-filter-details {
+    width: 220px;
+    word-wrap: break-word;
+}
+
 .gd-attribute-filter-dropdown-button-title__next {
+    display: flex;
```

Comment:
Is this safe (does it look same even if the tooltip is not displayed)? Otherwise, it can break dashboard plugins.

Reply:
Have checked it without tooltip, didn't change anything.